backend/controller/observability/observability.go

```diff
@@ -0,0 +1,23 @@
+package observability
+
+import (
+	"go.opentelemetry.io/otel/attribute"
+)
+
+type metricAttributeBuilders struct {
+	moduleName      func(name string) attribute.KeyValue
+	featureName     func(name string) attribute.KeyValue
+	destinationVerb func(name string) attribute.KeyValue
+}
```

Comment:
This is a bit excessive, could just do:

```
func attrModuleName(name string) attribute.KeyValue {...}
func attrFeatureName(name string) attribute.KeyValue {...}
...
```

Otherwise, it's just more lines to change/maintain.

Reply:
switched to explicit functions in the other PR